addons/power/functions/fnc_getBatteryLevel.sqf

```diff
@@ -0,0 +1,19 @@
+/**
+ * Get battery charging level. Returns as Wh and Percent.
+ *
+ * Arguments:
+ * 0: Battery <OBJECT>
+ * 
+ * Returns:
+ * 0: Battery Level <FLOAT>
+ * 1: Batttery Level Percent <FLOAT>
+ * 
+ */
+
+params ["_battery"];
+
+private _batteryLevel = _battery getVariable "AE3_power_batteryLevel";
```

Comment:
This will break, depending on if PR #185 is merged before or after this PR

Reply:
So, let's first merge your PR #185 and then I will try to fix that.